File: NumberGuess.java
import java.util.Random;
import java.util.Scanner;

public class NumberGuess{

    public static void main(String[]args){
    
    Scanner inputCollector= new Scanner(System.in);
    Random rand = new Random();
    


    int randomNumber = rand.nextInt(100);
    int userNumber = 0;
        
        for(int i =1; i<=5; i++){
        System.out.println("Enter Number");
        userNumber = inputCollector.nextInt();
            

            if(userNumber == randomNumber){
                System.out.println("Congrats chief ");

            }
            else if(userNumber < 0 || userNumber> 100 ){
                System.out.println("Warning! Enter number between 1-100");

            }
            else if(userNumber > randomNumber){
                System.out.println("The answer is lower than "+ userNumber);

            }
            else if(userNumber < randomNumber){
                System.out.println("The answer is higher than "+ userNumber);

            }



        }
        System.out.println("Coreect answer is " + randomNumber);

    }




}

File: number_guess.py
import random
a = random.randint(1, 101)
for i in range(1, 6):
    number= int(input("Enter number"))
    if(number == a):
        print("congratulations" , a ," is correct")
        break
    #elif(number !=a):
     #   print("Wrong guess")
    elif(number <0 or number >100):
        print("Warning! Enter number between 1 - 100")
    elif(number > a):
        print("The answer is lower than ", number, " chief!")
    elif(number < a):
        print("The answer is Higher than ", number, " chief!")

    

print("correct answer is ", a)
    



